persistence/msmq/index.md

```diff
@@ -1,10 +1,36 @@
 ---
-title: MSMQ Persistence
-summary: MSMQ Subscription Persistence for NServiceBus
+title: MSMQ Subscription Persistence
+summary: How to configure the MQMQ persistence for subscriptions
 component: MSMQPersistence
 reviewed: 2019-09-26
+related:
+ - samples/msmq/persistence
+redirects:
+ - persistence/msmq/subscription
 ---
 
 The MSMQ Subscription storage can be used to enable publish/subscribe with MSMQ without the need for any additional persister.
 
 NOTE: This can only be used to store subscriptions. To use sagas, timeouts, deferred messages, or the Outbox, a different persister is required.
+
+## Persistence at a glance
+
+|Feature                    |   |
+|:---                       |---
+|Supported Storage Types    |Subscriptions
+|Unsupported Storage Types  |Sagas, Outbox, Timeouts
+|Transactions               |Not supported
```

Comment:
The subscription message is transactionally persisted and that's all what this persister is about?